package.json

```diff
@@ -38,10 +38,10 @@
     "ajv": "^5.2.2",
     "cldr-data": "^31.0.2",
     "del": "^3.0.0",
-    "eslint": "^4.5.0",
+    "eslint": "^4.6.1",
```

Comment:
This will cause a conflict with the other PR (if it's merged before)

Reply:
Removed